src/core-styling/mixins/_scale.scss

```diff
@@ -0,0 +1,6 @@
+@use "../settings/font";
+@use "../../../node_modules/modularscale-sass/stylesheets/modularscale";
+
+@function scale($value: 0) {
+  @return round(modularscale.ms-function($value, font.$size, modularscale.$major-second));
```

Comment:
I would recommend defining the base size and the scale together (one can be changed, the other can't).

Reply:
Yup. I'm working on an improvement around baseline and scale, based on the concepts we discussed yesterday.